src/MICoreUnitTests/BasicLaunchOptionsTests.cs

```diff
@@ -263,7 +263,8 @@ internal void VerifyCoreApisPresent()
             launchOptions.WorkingDirectory = "/home/user";
             launchOptions.DebuggerMIMode = MIMode.Gdb;
             launchOptions.WaitDynamicLibLoad = false;
-            launchOptions.VisualizerFile = @"c:\myproject\file.natvis";
+            launchOptions.VisualizerFiles = new List<string>();
```

Comment:
@paulmaybee Do you know if the C++ team still owns something that relies on this property?

Reply:
One note is that we still support sending over this field as a string in JSON, just in the codebase everything is represented as a list so this had to change.